doc/changelog.rst

```diff
@@ -0,0 +1,65 @@
+Changelog
```

Comment:
i should have read this before the code...

Reply:
quite possibly